code/modules/client/preference/preferences.dm

```diff
@@ -642,7 +642,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 			continue
 		var/available_in_playtime = job.available_in_playtime(user.client)
 		if(available_in_playtime)
-			HTML += "<del>[rank]</del></td><td> \[ " + get_exp_format(available_in_playtime) + " as " + job.get_exp_req_type()  + " \]</td></tr>"
+			HTML += "<del><font color=red>[rank]</font></del></td><td><font color=red> \[ " + get_exp_format(available_in_playtime) + " as " + job.get_exp_req_type()  + " \]</font></td></tr>"
```

Comment:
This should probably use a css class, if anything, not "font color=red". 
Also, it should probably get rid of the "<del>", because that creates a white strikethrough which is still hard to read.

Perhaps simple black text with no (or black) strikethrough would be best.

Reply:
And how would I do that?

Reply:
Well, start by removing the del tags.
Then, see how it looks with no font color tags. Is the default text black? That might be okay.
If not, find the class in that HTML which sets the default color to white, and undo that so the default color is black.

Reply:
`HTML += "<span class="bad">[rank]</span></td><td><span class="bad"> \[ " + get_exp_format(available_in_playtime) + " as " + job.get_exp_req_type()  + " \]</span></td></tr>"`

The "bad" class from [this file](https://github.com/ParadiseSS13/Paradise/blob/add5787179fe220b0eddc4bd26099eb5a6761031/goon/browserassets/css/browserOutput.css) is red (#FF0000) with no other formatting.

Not sure what `<del>` does though.

Reply:
`<del>`does ~~strikethroughs~~

Reply:
Ohhh, I see. I looked to see if there was a more suitable span class but it actually seems that in this case we may have to create a new one in [this file](https://github.com/ParadiseSS13/Paradise/blob/add5787179fe220b0eddc4bd26099eb5a6761031/goon/browserassets/css/browserOutput.css)

Could do it like this:

`.not_available {color: #FF0000; text-decoration: line-through;}`
or however the spacing needs to be. I referred to documentation [here](https://www.electrictoolbox.com/css-strikethrough/) for the CSS information.